portfolio/src/main/java/com/google/sps/data/Comment.java

```diff
@@ -19,9 +19,11 @@
 
   private final long id;
   private final String text;
+  private final String useremail;
 
-  public Comment(long id, String text) {
+  public Comment(long id, String text, String useremail) {
```

Comment:
parameter and local variable names are in lowerCamelCase - `useremail` -> `userEmail`

https://google.github.io/styleguide/javaguide.html#s5.2.6-parameter-names